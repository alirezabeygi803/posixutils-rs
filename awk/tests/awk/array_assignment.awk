BEGIN {
    a[0] = 1
    print a[0]
    a[0] = "string"
    print a[0]
    a["other"] = "other string"
    print a["other"]
    print a[0]
    b["index"] = "test"
    print b["index"]
    print a[0]
    print a["other"]
}