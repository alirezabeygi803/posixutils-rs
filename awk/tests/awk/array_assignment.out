1
string
other string
string
test
string
other string
